server/stream.go

```diff
@@ -992,7 +992,7 @@ func (mset *stream) mirrorInfo() *StreamSourceInfo {
 	return mset.sourceInfo(mset.mirror)
 }
 
-const sourceHealthCheckInterval = 10 * time.Second
+const sourceHealthCheckInterval = 2 * time.Second
```

Comment:
I had changed that in my tests to not have to wait too long to reproduce the issue I was seeing. I would not have changed it otherwise.